bilingual-module/README

```diff
@@ -11,6 +11,7 @@ This is an Apertium language pair for translating between {{languageName1}} and
 For information on the latter two points, see subheading "For more
 information" below.
 
+
```

Comment:
Alas, let's not add too many newlines to these files. In fact, some Markdown linters would complain about this.